01-algorithms/02-algorithms-divide-conquer.md

```diff
@@ -45,205 +27,15 @@ QuickSort is an algorithm which takes a divide-and-conquer approach to sorting a
 1. Move all elements smaller than the pivot to the left and all elements larger than the pivot to the right. Note that the pivot is now in the correct index. Picking a pivot and rearranging the elements is referred to as _partitioning_.
 1. Perform QuickSort on the left and right sides of the pivot.
```

Comment:
I'm a visual learner so I feel like I'd benefit from adding the visualization back here